### ISSUE TITLE
Using a List as a value in a Map fails to compile

### ISSUE THREAD

**Issue by andrewdc** [sass/libsass#327]
tl:dr; If you use the correct format for a list (wrapped in parenthesis) as a value in a map - Libsass fails to compile.

Here is what I am trying to do:

``` sass
$properties: (
  font: (
    header: ("Ubuntu", sans-serif), 
    body:   ("Droid Sans", sans-serif),
    code:   ("Source Code Pro", sans-serif)
  ), 
//...etc...
```

Because a font-family is a comma separated list - I had asked @nex3 what the proper format should be, and he said I should wrap the list in (). That works in Ruby Sass.

Oddly, Libsass will compile this without (), without erroring. e.g.

``` sass
//this is the wrong format - but it works in Libsass, not Ruby Sass
header: "Ubuntu", sans-serif, 
```

Calling that value outputs: "Ubuntu", sans-serif  <-correctly without trailing comma

However, this fails to compile in Ruby Sass - which is what led me to ask what the correct format should be in the first place.
